lib/registrant.rb

```diff
@@ -0,0 +1,21 @@
+class Registrant
+    attr_accessor :name, :age, :permit, :license_data
```

Comment:
Careful here, we do not want `:permit` in the attr_reader since we never wanted to create a method called `permit`